src/arrayMethodSort.js

```diff
@@ -4,9 +4,41 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = defaultComparison) {
+    let cnt = 1;
```

Comment:
As for me `count` is more understandable, but maybe `cnt` is often practiced)

Reply:
agree, fix variable name, cnt - bad name